### ISSUE TITLE
Memory and other issues running browser

### ISSUE THREAD

**Issue by mddifilippo89** [IgniteUI/igniteui-react-examples#544]
@ChronosSF 
@dkamburov 

https://infragistics.visualstudio.com/NetAdvantage/_build/results?buildId=146832&view=logs&j=d3d782b6-b531-590c-3f8c-1d057b14221a&t=20fc2f5c-b8f0-5188-1a76-c13924e7d32a&l=228

**Comment by ChronosSF** [comment]
Yes, we know and we can't release the hgrid samples because of this currently. We are most likely hitting some limits to what is possible using react-app-rewired or create-react-app. With a lot of RAM and some pampering we can build it locally so we'll probably deploy manually. We will work on completely transferring the samples browser to vite as a long-term solution.

**Comment by mddifilippo89** [comment]
@turbobobbytraykov

**Comment by gmurray81** [comment]
> Yes, we know and we can't release the hgrid samples because of this currently. We are most likely hitting some limits to what is possible using react-app-rewired or create-react-app. With a lot of RAM and some pampering we can build it locally so we'll probably deploy manually. We will work on completely transferring the samples browser to vite as a long-term solution.

yeah, we've been meaning to move it to vite anyhow. We would just need to update the generation templates, and the code that glues the samples into the main browser possibly.

**Comment by gmurray81** [comment]
I HATE the way we did this in angular and extremely disagree that is a good strategy.

**Comment by gmurray81** [comment]
note, we also need to make sure that webpack is using the right entry point for the modules, it could be that it using the "flat" modules and tree shaking down might be using a lot more memory than if the entry points used by the bundler are the separated entry points.
